actors/builtin/market/market_state.go

```diff
@@ -231,7 +231,6 @@ func dealGetPaymentRemaining(deal *DealProposal, slashEpoch abi.ChainEpoch) abi.
 	}
 
 	durationRemaining := deal.EndEpoch - slashEpoch
-	Assert(durationRemaining > 0)
```

Comment:
Because `deal.StartEpoch` is passed in from outside, I don't think this assertion should be removed. I would just change it to `durationRemaining >= 0`